handbook/engineering/search/product.md

```diff
@@ -0,0 +1,29 @@
+# Search product team
+
+The search product team owns all parts of Sourcegraph that help users Compose search queries and navigate search results:
+  - Search field
+  - Search results UI
+  - Search contexts
+  - Query language, including structural search
```

Comment:
Missing the homepage panels and repogroup pages?

Reply:
thank you :)